src/CloudEvents/Serializer/JsonSerializer.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace CloudEvents\Serializer;
+
+use CloudEvents\CloudEventInterface;
+use CloudEvents\Serializer\Exception\UnsupportedVersionException;
+use CloudEvents\V1\CloudEventInterface as V1CloudEventInterface;
+use DateTimeInterface;
+
+use function array_filter;
+use function array_merge;
```

Comment:
These are overly verbose and can be removed.
```suggestion
```

Reply:
I did this because using the `use` statement here actually helps performance, as the PHP engine knows to check the root namespace for the function and skips checking the current namespace first. This was from PHP 7.1, but I don't think this behavior has changed at all, that I know of. https://veewee.github.io/blog/optimizing-php-performance-by-fq-function-calls/

Reply:
TIL 🙌 Apologies and thanks for the diligence here.